sw/vendor/riscv_compliance/riscv-target/opentitan/device/rv32imc/Makefile.include

```diff
@@ -26,8 +26,8 @@ else
 	# Verilator can be run in parallel mode so alter log output to prepend with
 	# test name, also increase uart_timeout as with parallel runs on CI individual
 	# test execution can take greater than 60 seconds.
-	PYTEST_OPT = --verilator_model $(TARGET_SIM) --test_bin $(<).32.vmem \
-		--rom_bin $(OT_BIN)/sw/device/boot_rom/boot_rom_sim_verilator.32.vmem \
+	PYTEST_OPT = --verilator_model $(TARGET_SIM) --test_bin $(<).64.vmem \
```

Comment:
feels like i should just change this to read-elf to avoid having the width impacting the command ... i'll probably do that as a separate PR, unless this does not work for some reason.

Reply:
forgot i need to make this a vendor patch...ugh.. i'll figure it out tomorrow.

Reply:
Oh god. Editing the riscv_compliance patches is neither easy nor fun. 

I currently have an open PR with a patch to update riscv_compliance, it would be good if this can wait until after it lands? (it is #2190).

Reply:
If you change anything there, switch from the VMEM to the ELF file, then you don't need to worry about the width any more.

Reply:
yeah that was my thinking too..but i think i need to make some edits with the pytest stuff that's being used, so I figured that would be better for another PR.  

Let me try to do that, and if this PR ends up merging after that, I'll make sure to switch to elf reads here.

@lenary 
I'm fine with waiting.  You can see I'm also gated on another ibex vendoring anyways, so I'm not in a rush.

Reply:
@lenary 
btw...since you are experiencing such pain :), did you want to write a quick readme or something for others to follow? Or is the issue not the process, just the fact that it needs patching? I did some late last year, but I remember very little about it...

Reply:
o this was my mistake, for some reason I thought pytest / memutil was hardcoded to use vmem. Turns out that's not the case. I'll do as you guys suggest.

Reply:
@tjaychen there are two issues. one is we don't have a way to generate concurrent patches of the riscv_compliance repo, sadly (so if someone is updating it in on PR, it cannot be updated in another).

The other is that editing `riscv_compliance` is not easy - most of the files have very cursed whitespace, which needs careful editing.